juniper_official/interface/check-optical-signal-loss-fec-tx-rx-power.rule

```diff
@@ -546,6 +546,13 @@ healthbot {
                                     }
                                 }								
                             }
+                            products PTX {
```

Comment:
seems like sensor "optical-sensor-oc-junos" is not used in any platform/product...is this correct?